src/libraries/System.Private.CoreLib/src/System/Random.cs

```diff
@@ -182,6 +182,113 @@ public virtual void NextBytes(byte[] buffer)
         /// <param name="buffer">The array to be filled with random numbers.</param>
         public virtual void NextBytes(Span<byte> buffer) => _impl.NextBytes(buffer);
 
+        /// <summary>
+        ///   Fills the elements of a specified span with items chosen at random from choices.
```

Comment:
```suggestion
        ///   Fills the elements of a specified span with items chosen at random from the provided set.
```

It may be a US English thing -- to me a choice means a decision, not one of a set of things to choose from, so this is more like item chosen from a population

Reply:
Eh, I may be wrong. It does jar for me but maybe usage changed?

https://dictionary.cambridge.org/dictionary/english/choice

Reply:
I am the last person on earth that should weigh in on English grammar 😅. That seems like an improvement to me, so I will address throughout.

Reply:
FWIW, I'd combine what was there and the suggestion to be "from the provided set of choices"